ContentFiles/Forms/FormsUIContent/App.xaml.pp

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Application
+    xmlns="http://xamarin.com/schemas/2014/forms"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    x:Class="$rootnamespace$.App">
```

Comment:
I think this namespace should be `FormsUI`, as that is assumed later when referencing this `App` class